doc/sphinx/dev_git_intro.rst

```diff
@@ -81,26 +81,42 @@ Pushing to your remote feature branch on your fork
 ----------------------------------------------------------
 When you are ready to push your updates to the remote feature branch on your fork
 
-1. type ``git status`` to list the file(s) that have been updated
+1. Type ``git status`` to list the file(s) that have been updated
 
-2. type ``git add <file>`` to add individual files, or ``git add --all`` to add all files, that have been updated to the staging area
-
-3. Commit the changes with ``git commit -m "your commit message"``. You can also type ``git commit`` to launch an editor in the terminal where you can enter your message.
-
-If you use the editor or BASH shell, you can easily break up your message over multiple lines for better readability.
-
-4. Push the updates to your fork: ``git push -u origin feature/[POD name]`` (The ``-u`` flag is for creating a new branch remotely and only needs to be used the first time.)
+2. Repeat steps 3--5 of section  (:ref:`ref-new-feature`)
 
 .. _ref-pull-request:
 
 Submitting Pull Requests
 ------------------------
-A Pull Request (PR) is your proposal to the maintainers to incorporate your feature into NOAA's repo. When your feature is ready, submit a PR by going to the GitHub page of your fork and clicking on **Pull request** to the right of the branch description. Make sure you are submitting the PR to NOAA-GFDL/develop. Enter a brief description for the PR, and check the boxes in the to-do list for the completed tasks. If you are still working on your POD, but want to test it with the CI, you can select the *Create Draft Pull Request* option from the dropdown menu by clicking the green button with the arrow to the right of the **Create Pull Request Button**.
-
+The pull request (PR) for your branch is your proposal to the maintainers to incorporate your feature into NOAA's repo.
```

Comment:
Should we mention the "draft PR" mode?  Developers could regularly practice these steps but the  review by the leads team won't occur until the draft status is removed.

Reply:
Ah - I see this below ... disregard!